packetbeat/docs/packetbeat-filtering.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[filtering-data]]
+=== Filtering the Exported Data
+
+include::../../libbeat/docs/filtering.asciidoc[]
+
+For example, the following filter configuration drops the `???` field from the exported document.
+
+// Can you help me provide a realistic example here?
```

Comment:
Here's a DNS example. It includes a subset of the Packetbeat fields so that just the requests and their response codes are reported.

```
filter:
  - include_fields:
      fields: [bytes_in, bytes_out, ip, client_ip, dns.question.name, dns.question.etld_plus_one, dns.response_code]
```

An alternative way of writing the list of fields:
```
filter:
  - include_fields:
      fields: 
        - bytes_in
        - bytes_out
        - ip
        - client_ip
        - dns.question.name
        - dns.question.etld_plus_one
        - dns.response_code
```

Filtered event:
```
{
  "@timestamp": "2016-03-28T14:48:21.732Z",
  "bytes_in": 32,
  "bytes_out": 48,
  "client_ip": "192.168.10.111",
  "dns": {
    "question": {
      "etld_plus_one": "google.com.",
      "name": "www.google.com."
    },
    "response_code": "NOERROR"
  },
  "ip": "8.8.8.8",
  "type": "dns"
}
```

Unfiltered event:
```
{
  "@timestamp": "2016-03-28T14:48:21.732Z",
  "beat": {
    "hostname": "macbook",
    "name": "macbook"
  },
  "bytes_in": 32,
  "bytes_out": 48,
  "client_ip": "192.168.10.111",
  "client_port": 50207,
  "client_proc": "",
  "client_server": "",
  "count": 1,
  "direction": "out",
  "dns": {
    "additionals_count": 0,
    "answers": [
      {
        "class": "IN",
        "data": "216.58.219.228",
        "name": "www.google.com.",
        "ttl": "165",
        "type": "A"
      }
    ],
    "answers_count": 1,
    "authorities_count": 0,
    "flags": {
      "authentic_data": false,
      "authoritative": false,
      "checking_disabled": false,
      "recursion_available": true,
      "recursion_desired": true,
      "truncated_response": false
    },
    "id": 6627,
    "op_code": "QUERY",
    "question": {
      "class": "IN",
      "etld_plus_one": "google.com.",
      "name": "www.google.com.",
      "type": "A"
    },
    "response_code": "NOERROR"
  },
  "ip": "8.8.8.8",
  "method": "QUERY",
  "port": 53,
  "proc": "",
  "query": "class IN, type A, www.google.com.",
  "resource": "www.google.com.",
  "responsetime": 1,
  "server": "",
  "status": "OK",
  "transport": "udp",
  "type": "dns"
}
```